src/test/java/com/dreamsportslabs/guardian/it/config/GetScopeIT.java

```diff
@@ -0,0 +1,524 @@
+package com.dreamsportslabs.guardian.it.config;
+
+import static com.dreamsportslabs.guardian.Constants.*;
+import static com.dreamsportslabs.guardian.utils.ApplicationIoUtils.*;
+import static org.apache.http.HttpStatus.*;
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.restassured.response.Response;
+import java.util.*;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class GetScopeIT {
+  @Test
+  @DisplayName("Should list scopes including created one")
+  public void testListScopesWithCreatedScope() {
+    // Arrange
+    String scopeName = RandomStringUtils.randomAlphabetic(10);
+    createScope(TENANT_1, valid(scopeName));
+
+    // Act
+    Response response = listScopes(TENANT_1, new HashMap<>());
+
+    // Validate
+    response.then().statusCode(SC_OK);
+    assertThat(response.jsonPath().getList("scopes.name"), hasItem(scopeName));
+
+    // Cleanup
+    deleteScope(TENANT_1, scopeName);
+  }
```

Comment:
This test should be written in a better way. 
The intention is to check all scopes are returned, not just the one created.